CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * Deprecations
     * Deprecated support for bash completion since this feature had slow performance. Also it relied on
     ``AutoCompleter`` which has since developed a dependency on ``cmd2`` methods. 
+    * Removed ability to call commands in ``pyscript`` as if they were functions (e.g ``app.help()``) in favor
```

Comment:
This should be under the **Potentially breaking changes** section since it could break a user's Python scripts.

Reply:
True, but it's also a deprecation which are always "Potentially breaking changes".
Still want it moved?

Reply:
I think in the past we have separated the categories as follows;
* Deprecations
    * This is where we recommend users stop using a feature because we plan to delete it in a future release
* Deletions (potentially breaking changes)
    * This is where a feature was deleted in the release and thus things which used it will break

Reply:
OK, I'll fix that